covalent_dispatcher/queue_consumer.py

```diff
@@ -52,7 +52,7 @@ def get_status():
 
 async def main():
     """Pick up workflows from the message queue and dispatch them one by one."""
-    nc = await nats.connect(MQ_CONNECTION_URI, connect_timeout=30)
+    nc = await nats.connect(MQ_CONNECTION_URI, reconnect_time_wait=1, max_reconnect_attempts=15)
```

Comment:
we're planning to replace NATS with SQS right?

Reply:
Yes, I think that work is in progress however.  Maybe it would be best to merge and create a separate PR to update the health check endpoints of the Queuer?